src/checks/ios.ts

```diff
@@ -323,7 +335,7 @@ async function validateSDKInitialization(project: iOSProject): Promise<void> {
   if (sdkInitializationFiles !== undefined) {
     logger.success(`iOS SDK Initialization found in ${sdkInitializationFiles}`);
   } else {
-    logger.warning('iOS SDK Initialization not found in suggested files');
+    logger.warning('iOS SDK Initialization not found');
```

Comment:
Can we name the files we check for?  I think its probably `AppDelegate` or something else because otherwise, this statement looks very definitive which shouldn't be the case because they could have a separate file for CIO-related operations.

Reply:
We can improve the error message if you have any suggestions in mind but mentioning exact files is difficult I think. Because we don't just check for exact names, we check for [specific keywords](https://github.com/customerio/cio-sdk-tools/blob/fcc68e00870b6ae2dc9af949ad8e1f798b7326dd/src/checks/ios.ts#L318) as well. When running on one of React Native sample, the output is

```
iOS SDK Initialization found in 'APN/ios/MyAppPushNotificationsHandler.swift', 'APN/ios/NotificationServiceExtension/NotificationServicePushHandler.swift'
```

Because the file names contain one of the keywords that we are looking for. So if they have a separate file for CIO related operations but the file name has any of the keyword that we have listed, the check doesn't fail and lists the file successfully.

Reply:
how about, 

> logger.warning('The iOS SDK Initialization may not be found in the expected files. For more details, run the command with to the "-d"')

and in the debuggable version, we print all the files we checked, and the pattern we used to check.

Reply:
That's a good idea. But our current `searchFilesRecursivelyForCode` method doesn't really know what exactly is it searching for, we probably need to add some info around it to explain the logs. Would you be okay if I merge this now and make a separate PR for the changes later?

Reply:
Yup sounds good 👍